src/screens/Followings/index.js

```diff
@@ -14,6 +14,7 @@ const Followings = ({dataFollower = [], isLoading, setDataFollower = () => {}})
         isLoadingDiscoveryUser={isLoading}
         showRecentSearch={true}
         withoutRecent={true}
+        isUser={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new prop `isUser` to the `<Followings>` component changes its external interface. Ensure that all instances of this component in the codebase are updated to handle this new prop. If this prop is optional, make sure it has a default value or the component handles the absence of this prop correctly.
```diff
+         isUser={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->